src/audio/drc/Testbench_drc_sin/init_sinedata_fixpt.h

```diff
@@ -0,0 +1,32 @@
+// SPDX - License - Identifier: BSD - 3 - Clause
+//
+//Copyright(c) 2020 Intel Corporation.All rights reserved.
+//
+//Author : Shriram Shastry <malladi.sastry@linux.intel.com>
+#pragma once
```

Comment:
The following codes behave the same with "#pragma once" and more common.
#ifndef DRC_SINE_FIXPT_H
￼#define DRC_SINE_FIXPT_H
...
#endif

However you should give it a name unique enough, e.g. INIT_SINADATA_FIXPT_H, TESTBENCH_DRC_SIN_INIT_SINEDATA_FIXPT_H

Reply:
OK